libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -254,7 +254,6 @@ - (void)send:(SFRestRequest *)request requestDelegate:(id<SFRestRequestDelegate>
             NSMutableDictionary *attributes = [[NSMutableDictionary alloc] init];
```

Comment:
Are these attributes still needed?

Reply:
ah, yeah.  Thanks for keeping me honest on these.  I'll remove these as I think to your point we don't need them.  I'll make sure we don't before I remove them.